openlattice.yaml

```diff
@@ -5590,6 +5767,198 @@ components:
           type: integer
         count:
           type: integer
+    Integration:
+      description: Represents a data integration, including all fields required to run the integration.
+      type: object
+      properties:
+        key:
+          description: a unique ID used for authorizing a call to run an integration
+          type: string
+          format: uuid
+        environment:
+          description: the retrofit environment (e.g. production, local)
```

Comment:
probably a good idea to change the example to `PROD_INTEGRATION`, `LOCAL` for clarity (/since you can't actually run an integration against the PRODUCTION environment)